DashAISoftware/DashAI
# Snackbar refactor

# Summary


This PR includes a refactoring that reduces code repetition when using `enqueueSnackbar`.

## Type of change



- Refactoring.

## Changes


- Modified `index.jsx` to establish default values for the snackbar properties
  - `anchorOrigin`: all snackbar messages are displayed at the top right corner.
  - `variant`: "error" is set as the default variant since the majority of snackbar messages are used to notify errors.
- All components that used `enqueueSnackbar` are modified to remove the specification of unnecesary properties.

## How to Test


1. Go to `DashAI/DashAI/front` and run `yarn build`
2. Go back to `DashAI/` and run `python -c "import DashAI;DashAI.run()"`
3. The snackbar notifications should work normally.
